docs/index.rst

```diff
@@ -5,65 +5,84 @@ Tahoe-LAFS
 .. Please view a nicely formatted version of this documentation at
    http://tahoe-lafs.readthedocs.io/en/latest/
 
-Contents:
+Tahoe-LAFS is a Free and Open decentralized cloud storage system.
+It distributes your data across multiple servers.
+Even if some of the servers fail or are taken over by an attacker,
+the entire file store continues to function correctly,
+preserving your privacy and security.
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
+   :caption: Preliminaries
```

Comment:
This section feels pretty weak.  I wonder if there's a way to get rid of it or flesh it out somewhat.

Reply:
How about just "Introduction"?

Reply:
Renamed the section to "Introduction" in 0f7e37d.